### ISSUE TITLE
'Needless blank line(s)' rule doesn't seem working nice

### ISSUE THREAD

**Issue by MateusGabi** [shyiko/ktlint#253]
Hello, KTLint print an error:
> 318:31 Needless blank line(s) (cannot be auto-corrected)

But there's no 'blank line' : 

![image](https://user-images.githubusercontent.com/14940643/42969677-f611725a-8b74-11e8-8f66-61c47b2be27a.png)

**Comment by shyiko** [comment]
Hi. Can you please provide a reproducible example (something I could `ktlint Example.kt`)? Thank you.

**Comment by MateusGabi** [comment]
I got it! This error just appears in the [original code](https://gist.github.com/MateusGabi/02eb7ca20f505527637a0869c9fe9f55). I've tried to remove useless lines of code. But if I remove these lines, the error just say goodbye :ghost:

**Comment by MateusGabi** [comment]
If you remove the last line, the error does not exist. Bingo! There's a bug in the line-counter.

Should be:
> 321:1 Needless blank line(s) (cannot be auto-corrected)

**Comment by shyiko** [comment]
You've got 2 newlines at the end of MainActivity.kt (which is why ktlint is complaining).
![img-2018-07-19-174256](https://user-images.githubusercontent.com/370176/42977331-68200cd4-8b7b-11e8-8c31-11ad623adca7.png)
(screenshot from Intellij IDEA)

The confusion is probably coming from the fact that in IDEA, vscode, etc. `}\n` is shown as 
```
}

```
while vim & gist.github.com show it as 

```
}
```

**Comment by MateusGabi** [comment]
Thank you :smile:  I'm sorry

**Comment by shyiko** [comment]
No worries! I wasn't aware of this quirk myself.
